src/overlays/actors/ovl_En_Ru1/z_en_ru1.h

```diff
@@ -34,7 +34,7 @@ typedef struct EnRu1 {
     /* 0x0286 */ s8 roomNum3;
     /* 0x0288 */ f32 unk_288;
     /* 0x028C */ BgBdanObjects* unk_28C;
-    /* 0x0290 */ s32 unk_290;
+    /* 0x0290 */ s32 preLimbDrawIndex; // should always be 0
```

Comment:
Dont think the comment is necessary personally. Sure in practice this is the case, because the array only has one function. But i dont think that fact needs to be explained in that way here.
A more helpful comment may be "indexs `sPreLimbDrawFuncs`" but even that i dont find necessary